### PR TITLE
[maliit-nemo-keyboard-git] add missing dependencies

### PR DESCRIPTION
**Pull request by jmlich**

### CODE REVIEW

File: maliit-nemo-keyboard-git/PKGBUILD [hunk 1]
```diff
@@ -10,12 +10,12 @@ pkgname=$_basename-git
...
 pkgver=0.99.2.r12.g7eb4c0fa
 
-pkgrel=1
+pkgrel=2
 pkgdesc="Contains the reference input method plugins, such as the Maliit Keyboard. "
 arch=('x86_64' 'aarch64')
 url="https://$_host/$_project/$_gitname#branch=$_branch"
 license=('BSD')
-depends=('maliit-framework')
+depends=('maliit-framework' 'maliit-keyboard' 'qt5-quickcontrols2')
```
Comment (neochapay) [inline maliit-nemo-keyboard-git/PKGBUILD:18]: maliit-keyboard not needed to nemo keyboard as i think - it's different keyboards

Comment (jmlich) [reply maliit-nemo-keyboard-git/PKGBUILD:18]: maliit-server complains on missing /usr/lib/maliit/plugins/libmaliit-keyboard-plugin.so when started manually. maliit-keyboard provides that file
```
pacman -F /usr/lib/maliit/plugins/libmaliit-keyboard-plugin.so
```

Comment (neochapay) [reply maliit-nemo-keyboard-git/PKGBUILD:18]: it's just warning, not error

Comment (jmlich) [reply maliit-nemo-keyboard-git/PKGBUILD]: Okay then. I have reverted that commit

**Pull request merged**